playground/app/routes/check-boxes.tsx

```diff
@@ -0,0 +1,111 @@
+import { conform, parse, useForm } from '@conform-to/react';
+import { type LoaderArgs, type ActionArgs, json } from '@remix-run/node';
+import { Form, useActionData, useLoaderData } from '@remix-run/react';
+import { useId } from 'react';
+import { Playground, Field } from '~/components';
+
+interface Schema {
+	answers: string[];
+}
+
+function parseForm(formData: FormData) {
+	return parse<Schema>(formData, {
+		resolve({ answers }) {
+			const error: Partial<Record<keyof Schema, string>> = {};
+
+			if (!answers?.length) {
+				error.answers = 'Required';
+			}
+
+			if (error.answers) {
+				return { error };
+			}
+
+			return {
+				value: {
+					answers,
+				},
+			};
+		},
+	});
+}
+
+export async function loader({ request }: LoaderArgs) {
+	const url = new URL(request.url);
+
+	return {
+		noClientValidate: url.searchParams.get('noClientValidate') === 'yes',
+	};
+}
+
+export async function action({ request }: ActionArgs) {
+	const formData = await request.formData();
+	const submission = parseForm(formData);
+
+	return json(submission);
+}
+
+export default function Example() {
+	const id = useId();
+	const { noClientValidate } = useLoaderData<typeof loader>();
+	const lastSubmission = useActionData<typeof action>();
+	const [form, { answers }] = useForm<Schema>({
+		id,
+		defaultValue: {
+			answers: [],
+		},
+		lastSubmission,
+		onValidate: !noClientValidate
+			? ({ formData }) => parseForm(formData)
+			: undefined,
+	});
+
+	return (
+		<Form method="post" {...form.props}>
+			<Playground title="Attributes" lastSubmission={lastSubmission}>
+				<Field label="Multiple Choice" config={answers}>
+					<label>
+						<input
+							{...conform.input(answers, {
+								type: 'checkbox',
+								value: 'a',
+								id: `${id}-answers-a`,
+							})}
+						/>
```

Comment:
Be aware that multiple inputs with the same name is not supported except for radio buttons. It will result in an array in the final payload and the error will be mapped to a path with the corresponding index (e.g. `answers[0]` instead of the original input name `answers`). Radio button is fine as it enforce only 1 value at the end instead.